### ISSUE TITLE
Add code command to PATH on remote session

### ISSUE THREAD

**Issue by kusnier** [microsoft/vscode#105557]
I am using a linux vm on my local machine do develop. My vscode is running directly on my local machine (windows).
It would be nice when the code command will be automatically available in the path of the terminal.

Actually I have to pick up the right version of the command. But it is not clear which of them is the right one.

![image](https://user-images.githubusercontent.com/726429/91526990-24385000-e905-11ea-9d1f-4caf16668475.png)

![image](https://user-images.githubusercontent.com/726429/91526721-90ff1a80-e904-11ea-9939-1a2d3d383342.png)

**Comment by aeschli** [comment]
Are you using Remote SSH to connect VSCode to the VM?

**Comment by kusnier** [comment]
> Are you using Remote SSH to connect VSCode to the VM?

Yes. My VM is managed by vagrant (Virtual box). So I am using Remote SSH to connect to it.
